w-books/src/App/components/Header/styles.css

```diff
@@ -0,0 +1,27 @@
+.Header {
```

Comment:
Use flex to styling the components. You shouldnt need use thinks like positions (relative/absolut)

Reply:
Use flex props like:
```
display: flex
flex-direction: row/column
justify-content: center/flex-start/flex-end/space-between/space-around
align-items: center/flex-start/flex-end
```